src/cljs/athens/views/athena.cljs

```diff
@@ -330,14 +339,14 @@
                                                    [:span.link-leader (use-sub-style result-style :link-leader) [(r/adapt-react-class Create)]]]
 
                                                   [:div (use-style result-style {:key      i
-                                                                                 :on-click (fn []
+                                                                                 :on-click (fn [e]
                                                                                              (let [selected-page {:node/title   title
                                                                                                                   :block/uid    uid
                                                                                                                   :block/string string
                                                                                                                   :query        query}]
                                                                                                (dispatch [:athena/toggle])
                                                                                                (dispatch [:athena/update-recent-items selected-page])
-                                                                                               (navigate-uid uid)))
+                                                                                               (open-in-main-or-sidebar e uid)))
```

Comment:
You can just use `navigate-uid` here, and pass the `e` parameter to it.

Reply:
oh, nice, I didn't check / know about that.